packages/npm/@amazeelabs/recipes/recipes/add-storybook.ts.md

```diff
@@ -0,0 +1,215 @@
+# Add storybook UI and Tailwind
+
+```typescript
+// Check node version.
+$$('node -v', {
+  stdout: $$.minimalVersion('12'),
+});
+
+// Check yarn version.
+$$('yarn -v', {
+  stdout: $$.minimalVersion('1.0'),
+});
+```
```

Comment:
In the other recipes i added this snippet to make sure we are inside a monorepo and also fetch the project name.
Then we also don't have  to prompt again later.

```suggestion
> This recipe has to be run from the root folder of a mono-repository created
> with `amazee-recipes create-monorepo`.

```typescript
$$('cat README.md', {
  stdout: /Executed `create-monorepo`/,
});

const { name: projectName } = $$.file('package.json');
```
```